script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -442,7 +453,7 @@ and those who came after are the young blood."
 [sha:d8625ee29f01cab9d5bf202efdc4daa9fa87a5bc]{
 -- Page 605, Offset 18092.
 -- 吸血鬼の尺度で言うと、ヴローヴはつい最近祖になった死徒である、と。
-From a vampire's perspective,
+From a vampire perspective,
```

Comment:
I think the possessive is more correct here

Reply:
Consider the difference between "from a human perspective" and "from a human's perspective"

I don't think I need to tell you which is more common. (the first)

Reply:
Hmm, it still feels a bit off. I feel like `human` may be a bit of a special case because it's both a noun and an _idea_ of sorts? Like if I replace human with something else
> From a fish perspective, 

it sounds totally busted. I'd still press for either possessive, or vampiric.

Reply:
Somehow it sounds fine to me anyways, but maybe it's just my busted senses.

I'll revert it